### ISSUE TITLE
Example from Getting Started not working

### ISSUE THREAD

**Issue by walter-weinmann** [JuliaLang/julia#42044]
### Windows 10  - Julia version 1.6.2

Example from https://docs.julialang.org/en/v1/manual/getting-started/ not working:

```
D:\SoftDevelopment\Projects\ora_bench>julia -e 'println(PROGRAM_FILE); for x in ARGS; println(x); end' foo bar
ERROR: syntax: incomplete: invalid character literal
Stacktrace:
 [1] top-level scope
   @ none:1

```

**Comment by Seelengrab** [comment]
Seems to be windows only, more minimal reproducer:

```
C:\WINDOWS>julia -e 'PROGRAM_FILE'                           
ERROR: syntax: character literal contains multiple characters
Stacktrace:                                                  
 [1] top-level scope                                         
   @ none:1                                                  
```

Works correctly with `"` instead of `'`:

```
C:\WINDOWS>julia -e "PROGRAM_FILE"
                                  
```

```
C:\WINDOWS>julia -e "println(PROGRAM_FILE); for x in ARGS; println(x); end" foo bar
                                                                                   
foo                                                                                
bar                                                                                
```

**Comment by mgkuhn** [comment]
See also both `Base.escape_microsoft_c_args` and `Base.shell_escape_wincmd` for how to prepare command-line arguments to be passed on fully transparently to `julia.exe` via `cmd.exe`, which works *very* differently from the quoting conventions of the POSIX shell.

```
julia> print(Base.shell_escape_wincmd(Base.escape_microsoft_c_args("julia",
       "-e", "println(PROGRAM_FILE); for x in ARGS; println(x); end", "foo", "bar")))
julia -e "println(PROGRAM_FILE); for x in ARGS; println(x); end" foo bar
```

**Comment by ViralBShah** [comment]
Many of the Getting started examples are unix specific, and perhaps this page should be updated to be more OS neural.

**Comment by mgkuhn** [comment]
OS neutral command-line examples are probably not that helpful as soon as examples get a bit more non-trivial, especially when passing on Julia string syntax with `-e`. The syntax (especially quoting of quotes) and even concepts (how does splitting the command-line into ARGS elements work and who does which part there) differ wildly between the Unix and Windows shells (and even between CMD.EXE and PowerShell!).

It sometimes can make more sense to provide both a POSIX shell *and* a CMD.EXE or a PowerShell prompt example, to raise awareness that there can be huge shell-specific differences when it comes to passing Julia syntax on the command-line. I think a better guideline would be to test all getting-started examples on
* bash 3 (for traditionalist macOS users still baulking at Apple's GPLv3 aversion)
* zsh (for newer macOS users)
* cmd.exe (for traditionalist Windows users)
* PowerShell 5.1 (for novelty-seeking Windows users)

and provide multiple examples in cases where a different syntax is more convenient.

**Comment by ViralBShah** [comment]
Right, but maybe the Getting started should de-emphasize the command line operation and stay with things that work everywhere?
